.github/workflows/testflight.yml

```diff
@@ -14,6 +14,7 @@ on:
   pull_request:
     paths:
       - '.github/workflows/testflight.yml'
+  workflow_dispatch:
```

Comment:
Is this a vestige of some test/experimentation?
```suggestion
```

Reply:
Not really, it allows us to trigger the uploading to Testflight directly from the GitHub web UI as soon as the line is available on main.